.github/dependabot.yml

```diff
@@ -13,8 +13,6 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 3
-    exclude-paths:
-      - "integration-tests/**"
```

Comment:
Was this causing an issue? @jackshirazi

Reply:
The byte buddy update would have passed if the integration tests are ignored, but it's a real problem and only the integration tests are showing that. So best we keep those tests in for upgrades